tools/deploy/etc/shrc.local

```diff
@@ -84,8 +84,6 @@ fi
 #DEBUG_DEV=1 - enable in shrc.custom
 
 if [ "${DEBUG}" = "1" -o "${DEBUG_DEV}" = "1" ]; then
-  JAVA_OPTS="${JAVA_OPTS} -Xdebug"
```

Comment:
these have been deprecated since like java 1.6 and -Xnoagent errors on newer java versions